File: 프로그래머스/0/181837. 커피 심부름/커피 심부름.cs
using System;

public class Solution 
{
    public int solution(string[] order) 
    {     
        int americanoCount = 0;
        int cafelatteCount = 0;
        for(int i=0; i<order.Length; i++)
        {
            if(order[i].Contains("americano") || order[i].Contains("anything"))
            {
                americanoCount++; 
            }
            else if(order[i].Contains("cafelatte"))
            {
                cafelatteCount++;
            }
        }
        return americanoCount * 4500 + cafelatteCount * 5000;
    }
}

File: 프로그래머스/0/181837. 커피 심부름/커피 심부름.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int solution(vector<string> order) 
{
    string am = "americano";
    string cf = "cafelatte";
    string at = "anything";
    
    int americanoCount = 0;
    int cafelatteCount = 0;
    for(string &s : order)
    {
        if(search(s.begin(), s.end(), am.begin(), am.end()) != s.end() ||
           search(s.begin(), s.end(), at.begin(), at.end()) != s.end())
        {
            americanoCount++;
        }
        else if(search(s.begin(), s.end(), cf.begin(), cf.end()) != s.end())
        {
            cafelatteCount++;
        }
    }
    return americanoCount * 4500 + cafelatteCount * 5000;
}